src/main.c

```diff
@@ -2144,6 +2145,7 @@ static GOptionEntry entries_convert[] = {
 	{"signing-keyring", '\0', 0, G_OPTION_ARG_FILENAME, &signing_keyring, "verification keyring file", "PEMFILE"},
 	{"mksquashfs-args", '\0', 0, G_OPTION_ARG_STRING, &mksquashfs_args, "mksquashfs extra args", "ARGS"},
 	{"casync-args", '\0', 0, G_OPTION_ARG_STRING, &casync_args, "casync extra args", "ARGS"},
+	{"ignore-image", '\0', 0, G_OPTION_ARG_STRING_ARRAY, &convert_ignore, "ignore image during conversion", "SLOTCLASS"},
```

Comment:
It should be explicit that the slot class name is needed here.

Reply:
FTR: I haven't changed anything here, yet since I wasn't sure what's the best way to make it more explicit